deploy/stacks/cognito.py

```diff
@@ -26,6 +28,7 @@ def __init__(
         vpc=None,
         prod_sizing=False,
         internet_facing=True,
+        custom_waf_rules=None,
```

Comment:
I have been tracking down this parameter and it seems unused upstream. It is not defined neither in the backend stack nor in the cdk.json file. Should we fix it in this PR or should we open a separate issue for it?